ebean-core/src/main/java/io/ebeaninternal/api/BindParams.java

```diff
@@ -424,7 +420,15 @@ public int hashCode() {
 
     @Override
     public boolean equals(Object o) {
-      return o != null && (o == this || (o instanceof Param) && hashCode() == o.hashCode());
+      if (this == o) return true;
+      if (o == null || getClass() != o.getClass()) return false;
+      Param param = (Param) o;
+      return encryptionKey == param.encryptionKey && isInParam == param.isInParam && isOutParam == param.isOutParam
+        && type == param.type && Objects.equals(inValue, param.inValue);
+    }
+
+    void queryBindHash(BindHash hash) {
+      hash.update(isInParam).update(isOutParam).update(type).update(inValue);
```

Comment:
has `encryptionKey` also part of the bindHash?

Reply:
Looking again ...

Reply:
No. Removing it.